### ISSUE TITLE
Can't send any log entry to Papertrail

### ISSUE THREAD

**Issue by benrom** [larskrantz/logger_papertrail_backend#5]
I'm trying to use this library with a Phoenix app, I followed every step of the configuration described on the Readme, I can see that the LoggerPapertrailBackend app is running with the observer, it reports no error but still fails to send anything to my papertrail dashboard.
I tried to telnet my papertrail host+port on the same computer and I managed to write some stuff to the dashboard.
I created a new empty Elixir project (`mix new`), set the proper configuration and calling Logger.error: still nothing on my dashboard.
The error seems to come from this library.

Running Elixir 1.3.4 with OTP 19. The unit tests on this project run fine with that version.

Any idea?

**Comment by larskrantz** [comment]
Hi,
that's strange, we are running about 20 applications without problems (however no Phoenix-app yet).
I created an [example repository](https://github.com/larskrantz/logger_papertrail_test) that worked as expected in Elixir 1.3.4 with OTP 19.
Could you try that repository?

I found an error in the README, `:logger` must also be started (but application would crash if not).

cheers/Lars

**Comment by benrom** [comment]
Thanks for the feedback and for this example!!
You're right, it works!
I see now there was an issue within my simple test project, though this was not in my phoenix project.
Another point is that I'm running my phoenix app within an umbrella project...
I'll spend some more time to try to find the issue.

**Comment by benrom** [comment]
I've just tried with a clean new phoenix project, it worked.
It must be related to using an umbrella project to hold this phoenix app I guess.

**Comment by benrom** [comment]
I'm getting closer to a solution, I found that it only works if I set the config inside the top config.exs of my umbrella project, not in the config files of my child phoenix app where I first wanted to add papertrail.
Not a big problem in my case because I'll want to use this in most of the apps within this project.
I guess we can close this ticket then, just keep it as information for people using umbrella projects.
